frontend/src/App.jsx

```diff
@@ -37,11 +37,8 @@ function App() {
   useEffect(() => {
     agent.Config.get().then(response => setConfig(response.data));
     pullUser().finally(() => setLoading(false));
-  }, []);
-
-  useEffect(() => {
     document.title = config.appName ? config.appName : __APPNAME__;
-  }, [config]);
```

Comment:
could be risky if the config is updated from somewhere else !

Reply:
This line was removed. `document.title` here is set only when the App is mounted.